packages/venia-ui/lib/components/CartPage/PriceSummary/giftCardSummary.ee.js

```diff
@@ -37,16 +37,23 @@ const getGiftCards = (cards = []) => {
 export default props => {
     const classes = mergeClasses({}, props.classes);
     const cards = getGiftCards(props.data);
+    const { locale } = useIntl();
 
     return cards.value ? (
-        <Fragment>
-            <span className={classes.lineItemLabel}>
-                {'Gift Card(s) applied'}
-            </span>
-            <span className={classes.price}>
-                {'-'}
-                <Price value={cards.value} currencyCode={cards.currency} />
-            </span>
-        </Fragment>
+        <IntlProvider locale="en-US">
```

Comment:
Is this intentional? This is an ee.js file not a spec.js file

Reply:
You are right, it was done by accident. Fixed

Thank you